ftw/mail/utils.py

```diff
@@ -37,6 +38,16 @@ def safe_decode_header(value):
     if apple_partial_encoding:
         value = ''.join(apple_partial_encoding.groups())
 
+    # Fix up RFC 2047 encoded words separated by 'CRLF LWSP' (which is fine
+    # according to the RFC) by replacing the CRLF with a SPACE so
+    # decode_header can parse them correctly.
+    # This works around a bug in decode_header that has been fixed in 3.3.
+    # See http://bugs.python.org/issue4491 and its duplicate.
+    # Example:
+    # From: =?utf-8?Q?B=C3=A4rengraben?=\r\n <from@example.org>
+    if ENCODED_WORD_WITHOUT_LWSP.match(value):
+        value = re.sub(ENCODED_WORD_WITHOUT_LWSP, '\\1 \\3', value)
```

Comment:
@jone I probably could. But there's no extra space in the resulting final output (there's a test for that). For a space to actually show up in the decoded output (more than once), it would have to be part of the encoded word (it works similar to whitespace in HTML that way).